README.md

```diff
@@ -1,2 +1,2 @@
 # PyHusky
-Husky for python -- on going
+git clone --recursive https://github.com/Gyingguo/pyhusky.git
```

Comment:
Why put this here? I think the status is still on-going. BTW, `https://github.com/Gyingguo/pyhusky.git` is your own repo.